Sync `svg/painting/reftests` from WPT upstream

Ahmad-S792 opened the pull request:
#### e432d4044e6e1c73d09d2ca39b965c8dbdb2dba0
<pre>
Sync `svg/painting/reftests` from WPT upstream

<a href="https://bugs.webkit.org/show_bug.cgi?id=269467">https://bugs.webkit.org/show_bug.cgi?id=269467</a>

Reviewed by Tim Nguyen.

This PR is to sync `svg/painting/reftests` from WPT upstream:

Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/b9dc4df0c6bdf28698ba6eecda23a5984e79b70e">https://github.com/web-platform-tests/wpt/commit/b9dc4df0c6bdf28698ba6eecda23a5984e79b70e</a>

&gt; Remove Test Expectations (.txt):
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-path-001-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-path-003-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-path-011-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-path-012-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-path-013-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-units-strokewidth-non-scaling-stroke-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-units-userspaceonuse-non-scaling-stroke-expected.txt:

&gt; Modified Files:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/w3c-import.log:
* LayoutTests/imported/w3c/resources/resource-files.json:

&gt; Modified Tests &amp; Test Expectations:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/display-none-mask-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/display-none-mask.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-path-001.svg:

&gt; New Tests &amp; Test Expectations:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/display-none-mask-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/fallback-001-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/fallback-001.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/fallback-002.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/fallback-002-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-implicit-subpaths-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-implicit-subpaths.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-path-001-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-path-002-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-path-003-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-path-011-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-path-012-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-path-013-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-path-021-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-path-022-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-path-023-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-units-strokewidth-non-scaling-stroke-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-units-userspaceonuse-non-scaling-stroke-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/markers-orient-001-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/markers-orient-002-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/markers-orient-002.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/mask-percentage-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/mask-percentage-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/mask-percentage.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/mask-print-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/paint-context-001-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/paint-context-002-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/paint-order-001-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/paintorder-text-decorations-ref.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/paintorder-text-decorations.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/percentage-attribute-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/percentage-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/paintorder-text-decorations-expected.svg:

&gt; Test Expectations [ Failure ]:
* LayoutTests/TestExpectations:

&gt; Test Expectations [ Failure &amp; ImageOnlyFailure] - Platform:
* LayoutTests/platform/ios/TestExpectations:
* LayoutTests/platform/mac/TestExpectations:

Canonical link: <a href="https://commits.webkit.org/274810@main">https://commits.webkit.org/274810@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/a57cf661682b3fb94fa4536e63872155b91b6a0b

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/40066 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/19078 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/42442 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/42611 "Built successfully") | [  ~~🛠 wincairo~~](https://ews-build.webkit.org/#/builders/32/builds/36050 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/21987 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/16407 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/33337 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/40640 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/16059 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/34605 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/13893 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/13921 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/35557 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/43889 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/36347 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/35893 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/39656 "Passed tests") | 
| | [  ~~🛠 tv~~](https://ews-build.webkit.org/#/builders/44/builds/14881 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/12214 "Passed tests") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/21/builds/37923 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/16500 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/34839 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/8995 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/16549 "Built successfully") | | | 
| | [  ~~🛠 watch-sim~~](https://ews-build.webkit.org/#/builders/45/builds/16144 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | | 
<!--EWS-Status-Bubble-End-->

nt1m commented on LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/paintorder-text-decorations-ref.svg:
The -expected.svg didn't get created for this, because of https://github.com/web-platform-tests/wpt/pull/44610 , so this won't get run in WebKitTestRunner. Can you run the import script again once that lands (and change the upstream commit hash)?

Ahmad-S792 replied:
Will do once upstream will merge. :-)

Ahmad-S792 replied:
Pushed in next one with updated commit hash.

Ahmad-S792 committed:
Sync `svg/painting/reftests` from WPT upstream

https://bugs.webkit.org/show_bug.cgi?id=269467

Reviewed by Tim Nguyen.

This PR is to sync `svg/painting/reftests` from WPT upstream:

Upstream commit: https://github.com/web-platform-tests/wpt/commit/b9dc4df0c6bdf28698ba6eecda23a5984e79b70e

> Remove Test Expectations (.txt):
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-path-001-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-path-003-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-path-011-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-path-012-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-path-013-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-units-strokewidth-non-scaling-stroke-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-units-userspaceonuse-non-scaling-stroke-expected.txt:

> Modified Files:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/w3c-import.log:
* LayoutTests/imported/w3c/resources/resource-files.json:

> Modified Tests & Test Expectations:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/display-none-mask-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/display-none-mask.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-path-001.svg:

> New Tests & Test Expectations:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/display-none-mask-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/fallback-001-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/fallback-001.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/fallback-002.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/fallback-002-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-implicit-subpaths-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-implicit-subpaths.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-path-001-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-path-002-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-path-003-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-path-011-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-path-012-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-path-013-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-path-021-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-path-022-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-path-023-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-units-strokewidth-non-scaling-stroke-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/marker-units-userspaceonuse-non-scaling-stroke-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/markers-orient-001-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/markers-orient-002-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/markers-orient-002.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/mask-percentage-expected.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/mask-percentage-ref.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/mask-percentage.html:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/mask-print-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/paint-context-001-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/paint-context-002-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/paint-order-001-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/paintorder-text-decorations-ref.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/paintorder-text-decorations.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/percentage-attribute-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/percentage-expected.svg:
* LayoutTests/imported/w3c/web-platform-tests/svg/painting/reftests/paintorder-text-decorations-expected.svg:

> Test Expectations [ Failure ]:
* LayoutTests/TestExpectations:

> Test Expectations [ Failure & ImageOnlyFailure] - Platform:
* LayoutTests/platform/ios/TestExpectations:
* LayoutTests/platform/mac/TestExpectations:

Canonical link: https://commits.webkit.org/274810@main
